src/preferences/dialog/dlgprefreplaygaindlg.ui

```diff
@@ -246,6 +221,43 @@
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="groupBoxAnalyzis">
+     <property name="title">
+      <string>Analyzis</string>
```

Comment:
"To analyze" is the correct spelling of the verb, but "analysis" is the correct spelling of that noun. Yes, English is strange.